### DISCUSSION TITLE
Automatic1111 v1.10 with SD3

### DISCUSSION THREAD

**Discussion by phr0gz** [AbdBarho/stable-diffusion-webui-docker#716] [Q&A] [answered]
Anyone have it working ? If yes can you be kind and share a PR or just what need to be changed?

**Comment by slix** [comment]
https://github.com/AUTOMATIC1111/stable-diffusion-webui/releases

Now at 1.10.1 for a bug fix

**Comment by ghost** [comment, marked as answer]
I used the directions mentioned here [https://github.com/AbdBarho/stable-diffusion-webui-docker/discussions/672](https://github.com/AbdBarho/stable-diffusion-webui-docker/discussions/672)

Version number updated, so i assume it was successful.  Can't validate if anything is broken, but everything looks fine.

**Reply by phr0gz** [reply]
it's working thx, I was able to get some picture with SD3. I didn't test if the extensions are working.

**Comment by SachiaLanlus** [comment]
I have reviewed the changelog.
There is nothing new to be adjusted. So you can just replace the tag to update.
But there is a caveat if you installed controlnet plugin, it might cause some dependency conflict between the main framework.
The controlnet want to use the latest mediapipe package which install the latest protobuf package.
But the Automatic1111 v1.10 state that the desired protobuf version is 3.20.0.
I do not know if it can run on the version newer than 3.20.0.
So if you encounter any issue about controlnet. You can try to uninstall mediapipe and reinstall the 3.20.0 protobuf and install the mediapipe again.
You can execute the command below in the container shell.
```
pip uninstall mediapipe
pip install "protobuf==3.20.0" mediapipe
```

**Comment by SachiaLanlus** [comment]
Oh by the way
I am also considering that do we need to include the SD3 related setting or model to be placed inside the image like what we did to the SD1.5.
But it will take up your storage much more.
